frontend/src/components/projects/ProjectsAssociatedSamples.js

```diff
@@ -0,0 +1,155 @@
+import React, {useEffect} from "react";
+import {connect} from "react-redux";
+import {Link} from "react-router-dom";
+import {Button, Tag} from "antd";
+
+import PageContent from "../PageContent";
+import PaginatedTable from "../PaginatedTable";
+
+import api, {withToken}  from "../../utils/api"
+
+import {listTable, setFilter, setFilterOption, clearFilters, setSortBy} from "../../modules/samples/actions";
+import filterSamplesByProject from "../../utils/filterSamplesByProject";
+import {SAMPLE_FILTERS} from "../filters/descriptions";
+import getFilterProps from "../filters/getFilterProps";
+import getNFilters from "../filters/getNFilters";
+import FiltersWarning from "../filters/FiltersWarning";
+import {SampleDepletion} from "../samples/SampleDepletion";
+import mergedListQueryParams from "../../utils/mergedListQueryParams";
+
+const getTableColumns = (sampleKinds) => [
+    {
+      title: "Sample Kind",
+      dataIndex: "sample_kind__name",
+      sorter: true,
+      width: 70,
+      options: sampleKinds.items.map(x => ({ label: x.name, value: x.name })), // for getFilterProps
+      render: (_, sample) =>
+        <Tag>{sampleKinds.itemsByID[sample.sample_kind].name}</Tag>,
+    },
+    {
+      title: "Name",
+      dataIndex: "name",
+      sorter: true,
+      width: 170,
+      render: (name, sample) =>
+        <Link to={`/samples/${sample.id}`}>
+          <div>{name}</div>
+          {sample.alias &&
+            <div><small>alias: {sample.alias}</small></div>
+          }
+        </Link>,
+    },
+    {
+      title: "Cohort",
+      dataIndex: "cohort",
```

Comment:
'Cohort' attribute is on an Individual, not a Sample. Therefore, it will be listed as empty even if your sample is associated to an individual which does have a cohort.

i.e:
![image](https://user-images.githubusercontent.com/1617741/133156560-a7d472b6-456d-4cad-b401-0da9777b479c.png)

![image](https://user-images.githubusercontent.com/1617741/133156630-0750740d-19da-4a80-b9ed-9e5349760afa.png)

![image](https://user-images.githubusercontent.com/1617741/133156668-0da745b8-6022-4b92-84ea-25c019417039.png)

Reply:
dataIndex would be individual__cohort ?